src/Symfony/Component/Intl/Tests/CountriesWithUserAssignedTest.php

```diff
@@ -0,0 +1,1037 @@
+        'MD' => '498',
+        'ME' => '499',
+        'MF' => '663',
+        'MG' => '450',
+        'MH' => '584',
+        'MK' => '807',
+        'ML' => '466',
+        'MM' => '104',
+        'MN' => '496',
+        'MO' => '446',
+        'MP' => '580',
+        'MQ' => '474',
+        'MR' => '478',
+        'MS' => '500',
+        'MT' => '470',
+        'MU' => '480',
+        'MV' => '462',
+        'MW' => '454',
+        'MX' => '484',
+        'MY' => '458',
+        'MZ' => '508',
+        'NA' => '516',
+        'NC' => '540',
+        'NE' => '562',
+        'NF' => '574',
+        'NG' => '566',
+        'NI' => '558',
+        'NL' => '528',
+        'NO' => '578',
+        'NP' => '524',
+        'NR' => '520',
+        'NU' => '570',
+        'NZ' => '554',
+        'OM' => '512',
+        'PA' => '591',
+        'PE' => '604',
+        'PF' => '258',
+        'PG' => '598',
+        'PH' => '608',
+        'PK' => '586',
+        'PL' => '616',
+        'PM' => '666',
+        'PN' => '612',
+        'PR' => '630',
+        'PS' => '275',
+        'PT' => '620',
+        'PW' => '585',
+        'PY' => '600',
+        'QA' => '634',
+        'RE' => '638',
+        'RO' => '642',
+        'RS' => '688',
+        'RU' => '643',
+        'RW' => '646',
+        'SA' => '682',
+        'SB' => '090',
+        'SC' => '690',
+        'SD' => '729',
+        'SE' => '752',
+        'SG' => '702',
+        'SH' => '654',
+        'SI' => '705',
+        'SJ' => '744',
+        'SK' => '703',
+        'SL' => '694',
+        'SM' => '674',
+        'SN' => '686',
+        'SO' => '706',
+        'SR' => '740',
+        'SS' => '728',
+        'ST' => '678',
+        'SV' => '222',
+        'SX' => '534',
+        'SY' => '760',
+        'SZ' => '748',
+        'TC' => '796',
+        'TD' => '148',
+        'TF' => '260',
+        'TG' => '768',
+        'TH' => '764',
+        'TJ' => '762',
+        'TK' => '772',
+        'TL' => '626',
+        'TM' => '795',
+        'TN' => '788',
+        'TO' => '776',
+        'TR' => '792',
+        'TT' => '780',
+        'TV' => '798',
+        'TW' => '158',
+        'TZ' => '834',
+        'UA' => '804',
+        'UG' => '800',
+        'UM' => '581',
+        'US' => '840',
+        'UY' => '858',
+        'UZ' => '860',
+        'VA' => '336',
+        'VC' => '670',
+        'VE' => '862',
+        'VG' => '092',
+        'VI' => '850',
+        'VN' => '704',
+        'VU' => '548',
+        'WF' => '876',
+        'WS' => '882',
+        'YE' => '887',
+        'YT' => '175',
+        'ZA' => '710',
+        'ZM' => '894',
+        'ZW' => '716',
+        'XK' => '983',
+    ];
+
+    public static function setUpBeforeClass(): void
+    {
+        // other tests might or might not call Countries first and set SYMFONY_INTL_ALLOW_USER_ASSIGNED
+        $ref = new \ReflectionProperty(Countries::class, 'allowUserAssigned');
+        $ref->setValue(null, true);
```

Comment:
why not just set the env var (`putenv`) to enable the new feature

Reply:
Because there are other tests that run before this one, and they set it to false permanently